src/controllers/tripsController.js

```diff
@@ -138,7 +138,16 @@ export default class requestController {
           passportName: passportName.toLowerCase().trim(),
           role
         }],
-      }, { where: { id } });
+      }, { where: { id }, returning: true, plain: true });
+      const manager = await db.User.findOne({ where: { id: updatedRequest[1].managerId } });
+      const notification = await notifService.createNotif(manager.id, manager.email, `Request with id ${updatedRequest[1].id} has been edited`, '#');
+      const content = {
+        intro: req.__('request with id %s has been edited', updatedRequest[1].id),
+        instruction: req.__('To view this edited request click below'),
+        text: req.__('View request'),
+        signature: req.__('signature')
+      };
+      await SendNotification.sendNotif(notification, req, content);
       return Response.success(res, 200, res.__('Request updated successfully'), updatedRequest);
     } catch (err) {
       return Response.errorResponse(res, 500, err.message);
```

Comment:
could you change this to 'server error' instead of err.message so that we don't continue to have it on develop?

Reply:
I changed it